wiki/Tournaments/OWC/2023/en.md

```diff
@@ -144,12 +198,78 @@ The complete sign-up list can be found [here](https://gist.github.com/LeoFLT/82e
   2. [A-One feat. Shihori - Bamboo Dance (nanoya) \[Lunatic\]](https://osu.ppy.sh/beatmapsets/2075280#osu/4344475)
   3. [Sable Hills - Not Falling (squirrelpascals) \[Vid\]](https://osu.ppy.sh/beatmapsets/2075253#osu/4344422)
 
+## Results
+
+### Qualifiers
+
+The final standings for the Qualifier stage can be found in the following [spreadsheet](https://docs.google.com/spreadsheets/d/1iih6qvAYpCakmdZcnFoB_XGxBm4WDBYNNeo-81PcKGc?rm=minimal).
+
+| Seed | Country | %MAX sum[^qualifiers-seeding] | avg. score[^qualifiers-tiebreaker] |
+| :-: | :-- | :-- | :-- |
```

Comment:
for readability of this table--
- right align the last two columns
- *don't* left-pad numbers (average scores of Croatia and India)
- right-pad numbers to use the same precision (e.g. poland %max should read `10.330`)

Reply:
🤦 I forgot to apply the correct formatting to the last two tables. Applied all suggested fixes.